docker-compose.dev.yml

```diff
@@ -14,7 +14,7 @@ services:
     networks:
       - gfw-data-api
     environment:
-      - DB_HOST_RO=database
+      - DB_HOST_RO=database_12
```

Comment:
did you mean to include this here, or is this just your local db name?

Reply:
I was working against `dev` branch of `gfw-data-api` that's pointing to that name (from postgres 12 upgrade Thomas pushed).